### PR TITLE
#164047023 implement reset password

### PR DESCRIPTION
**Pull request by winniekariuki**
## Description ##
This pull request enables the user to reset their password when they forget.

## Type of change ##
Please select the relevant option
- [x] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [x] Integration test
- [ ] unit test

## Checklist: ##
- [ ] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[164047023](https://www.pivotaltracker.com/story/show/164047023)

## Reviewers ##
- @Derrickkip 
- @Kibetchirchir 
- @JumaKahiga 

## Relevant screenshots: ##
Screenshot 2019-04-15 at 11 13 27
Screenshot 2019-04-15 at 11 14 44

### CODE REVIEW

File: src/__tests__/RequestPassword.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import {
+  RequestResetForm,
+  mapDispatchToProps,
+  mapStateToProps,
+} from '../components/ResetPassword/requestReset';
+
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (Derrickkip) [inline src/__tests__/RequestPassword.test.js]: Enzyme is already configured in setUpTests.js

File: src/__tests__/ResetPassword.test.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import ResetPassword from '../components/ResetPassword/resetPassword';
+
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (Derrickkip) [inline src/__tests__/ResetPassword.test.js]: Same as above

**Comment by Derrickkip** [comment]
When I enter an email that is not valid, this is what I get

Screenshot 2019-04-11 at 16 25 22

**Comment by winniekariuki** [comment]
am also fixing the bug thanks for the review I will inform you once am done for another  review

**Comment by winniekariuki** [comment]
I have fixed the issue on email that is not registered kindly review the changes as I fix the tests issues and hound

File: src/components/ResetPassword/requestReset.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            <div>
+              <div>
+                <p className="message">
+                  {this.state.message ? this.state.message : null}
+                </p>
+
+                {this.state.errors ? (
+                  this.state.errors.includes('If we found an account') ? (
+                    <p className="message">
+                      If we found an account; we've emailed you a link to change
```
Comment (mnswaleh) [inline src/components/ResetPassword/requestReset.js:50]: use `&#39;` for `'` to escape the single quote

File: src/__tests__/__snapshots__/ResetPassword.test.js.snap [hunk 4]
```diff
@@ -0,0 +1,57 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (mnswaleh) [inline src/__tests__/__snapshots__/ResetPassword.test.js.snap]: gitignore the snapshots

**Review by kevpy — APPROVED** [review]

**Review by kevpy — APPROVED** [review]

File: src/components/signup/register.scss [hunk 5]
```diff
@@ -1,4 +1,12 @@
-@import "../../css/variables";
+@import '../../css/variables';
+$theme: #27586e;
+$white: #fff;
+$theme: #27586e;
+$black: #0a0a0a;
+$danger: #fc9a94;
+$green: #519e8a;
+$shadow-white: #cfcfcf;
+$shadow: #555;
```
Comment (mnswaleh) [inline src/components/signup/register.scss]: These variables are not used anywhere in the code

File: src/__tests__/RequestPassword.test.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```
Comment (mnswaleh) [inline src/__tests__/RequestPassword.test.js:1]: Tests are supposed to be in the component folder

**Review by JumaKahiga — APPROVED** [review]

File: src/components/signup/register.scss (hunk 5 above)
Comment (winniekariuki) [reply src/components/signup/register.scss]: Changes made

File: src/__tests__/RequestPassword.test.js (hunk 6 above)
Comment (winniekariuki) [reply src/__tests__/RequestPassword.test.js:1]: Changes made

**Review by Kibetchirchir — APPROVED** [review]

**Review by Derrickkip — APPROVED** [review]

**Pull request merged**